src/main/java/io/ebean/dbmigration/migration/DdlScript.java

```diff
@@ -0,0 +1,73 @@
+package io.ebean.dbmigration.migration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import javax.xml.bind.annotation.XmlValue;
+
+
+/**
+ * <p>Java class for anonymous complex type.
+ * <p>
+ * <p>The following schema fragment specifies the expected content contained within this class.
+ * <p>
+ * <pre>
+ * TODO @Rob: Can this generated automatically?
```

Comment:
@rbygrave how did you generate that documentation?

Reply:
XML tools as part of IntelliJ IDEA